tests/album-read.test.js

```diff
@@ -0,0 +1,78 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Read Albums', () => {
+  let artists;
+  let albums;
+  beforeEach(async () => {
+    const responses = await Promise.all([
+      db.query(
+        'INSERT INTO Artists (id, name, genre) VALUES( $1, $2, $3) RETURNING *',
+        [1, 'Tame Impala', 'rock']
+      ),
+      db.query(
+        'INSERT INTO Artists (id, name, genre) VALUES( $1, $2, $3) RETURNING *',
+        [2, 'Kylie Minogue', 'pop']
+      ),
+      db.query(
+        'INSERT INTO Artists (id, name, genre) VALUES( $1, $2, $3) RETURNING *',
+        [3, 'Tame Antelope', 'jazz']
+      ),
+    ]);
+
+    artists = responses.map(({ rows }) => rows[0]);
+
+    const albums_responses = await Promise.all([
+      db.query(
+        'INSERT INTO Albums (id, artist_id, name, year) VALUES ($1, $2, $3, $4) RETURNING *',
+        [1, artists[0].id, 'Currents', 2015]
+      ),
+      db.query(
+        'INSERT INTO Albums (id, artist_id, name, year) VALUES ($1, $2, $3, $4) RETURNING *',
+        [2, artists[1].id, 'Light Years', 2000]
+      ),
+      db.query(
+        'INSERT INTO Albums (id, artist_id, name, year) VALUES ($1, $2, $3, $4) RETURNING *',
+        [3, artists[2].id, 'Random Name', 2015]
+      ),
+    ]);
+
+    albums = albums_responses.map(({ rows }) => rows[0]);
+  });
+
+  describe('GET /albums', () => {
+    it('returns all album records in the database', async () => {
+      const { status, body } = await request(app).get('/albums').send();
+
+      expect(status).to.equal(200);
+      expect(body.length).to.equal(3);
+
+      body.forEach((albumRecord) => {
+        const expected = albums.find((a) => a.id === albumRecord.id);
+
+        expect(albumRecord).to.deep.equal(expected);
+      });
+    });
+  });
+  describe('GET /albums/{id}', () => {
```

Comment:
I would add a clear line here for consistency as you have clear lines between tests and the beforeEach and describe block